### ISSUE TITLE
Android 12 support

### ISSUE THREAD

**Issue by billnbell** [zendesk/sunshine-conversations-android#166]
Can you add for Android 12?
```
smooch:core:8.2.0 manifest

<service android:name="io.smooch.core.FcmService" >
            <intent-filter>
                <action android:name="com.google.firebase.MESSAGING_EVENT" />
            </intent-filter>
        </service>
Switch to add android:exported="false"?

<service android:name="io.smooch.core.FcmService"  android:exported="false" >
            <intent-filter>
                <action android:name="com.google.firebase.MESSAGING_EVENT" />
            </intent-filter>
        </service>
```

**Comment by DataWearsAHood** [comment]
In which file(s) would this change be applied to?
Which release/version are you basing this change request on?

**Comment by billnbell** [comment]
This happens with compileSdk is set to SDK 31. Open Android Studio and you can see the merged manifests (open AndroidManifest.xml, look at bottom of screen for merged tab).

It was missing in smooch:ui:9.0.2 and smooch:core:9.0.2 manifests

I think it may already be fixed ?

**Comment by billnbell** [comment]
Oh this was an issue on smooch:core:8.2.0 and seems to be fixed in 9.0.2. We were on 8.2.0 for a while.

Just open your smooch:core AndroidManifest.xml and fix it on 8.2.0... 

But we are good, since we upgraded.
